README.md

```diff
@@ -538,3 +539,23 @@ make helm-test # Helm unit test
 make kubeconform # Kubernetes manifest validation
 make checkcov # Check misconfigurations
 ```
+
+### Tracing
+
+Prometheus RDS Exporter includes an OpenTelemetry trace exporter to facilitate troubleshooting.
+
+Traces can be forwarded to any OpenTelemetry server using gRPC protocol.
+
+1. Export the `OTEL_EXPORTER_OTLP_ENDPOINT` variable.
+
+    See [OpenTelemetry environments variables](https://opentelemetry.io/docs/languages/sdk-configuration/otlp-exporter/).
+
+    ```bash
+    export OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4317
```

Comment:
Maybe add the service name as a variable as well?

For instance, we're sending everything to the same Tempo backend adding the environment to each service name to differentiate them

Reply:
It's actually possible with OTEL SDK built-in environment variables since we are using `resource.WithFromEnv()` [here](https://github.com/qonto/prometheus-rds-exporter/pull/118/files#diff-7c90a46b47814bfe95c9c4d5ec47d30178d010b30c7285a72c07031e9b9f35a2R65).

For instance, the following environment variable will export it as resource attribute:

```
export OTEL_RESOURCE_ATTRIBUTES=service.name=my-custome,service.key1=value1
```

<img width="722" alt="screenshot 2024-02-01 at 11 48 31" src="https://github.com/qonto/prometheus-rds-exporter/assets/319005/6942a449-c314-4f46-bc17-3b244e3129df">

Reply:
Thanks. 

It's up to you if you want to set this as a variable or not, actually. Even if we use attributes, it won't hurt to leave this as an option, but feel free to close this thread.

I would also want to split other services using attributes directly (even with `k8s_attributes` processor), but we're not there yet.